projectile.el

```diff
@@ -1779,6 +1780,11 @@ This is a subset of `grep-read-files', where either a matching entry from
                (and ext (concat "*." ext)))))
       (or default-alias default-extension))))
 
+(defun projectile-grep-ignored-patterns ()
```

Comment:
> I found projectile-patterns-to-ignore which returns list of .projectile entries without leading slash. But it doesn't used for grep ignoring. This is feature or bug?

Pretty sure it's used with `grep` but on a bit more higher level (several sources of ignored files get merged together there).



> May be better name would be projectile-grep-ignored or projectile-grep-find-ignored (as analog of projectile-project-ignored for grep/ag).

I guess you should check the exact code we have in `projectile-grep` right now. I'm fairly certain it respects ignored files.